_posts/2016-04-12-introducing-blog-bear.md

```diff
@@ -0,0 +1,27 @@
+---
+
+title: 'Meet Blog Bear'
+author: Jenn
+layout: blog-post
+comments: false
+
+---
+
+By now you’re probably familiar with our animal friends the [Low-Profile Dog](http://hood.ie/animals/#low-profile-dog) (or LPD, as some of us like to call him) and the [Docs Chicken](http://hood.ie/animals/#docs-chicken). These animal friends play a huge part in making the community fun and welcoming and we are so happy to have collaborated with the very talented [Carolina Buzio](https://twitter.com/carolinabuzio) on their creation.
+
+Please join us in welcoming our newest animal mascot to the Hoodieverse, captured perfectly by editorial contributor [Annika Howard](https://github.com/NeekyRabbit).
+
+# Meet our latest animal friend: Blog Bear
+
+<a href="http://hoodiehq.github.io/hoodie-css/src/content_img/animals/blog-bear-1.png></a>
```

Comment:
You're right, I spaced and added the tag while I was writing the copy and never updated oops!